src/Frontend/Components/ProgressBar/ProgressBar.tsx

```diff
@@ -49,31 +43,78 @@ export function ProgressBar(props: ProgressBarProps): ReactElement {
       ? resourcesWithCriticalExternalAttributions
       : props.progressBarData.resourcesWithNonInheritedExternalAttributionOnly,
   );
+  const data: Array<{ [key: string]: number }> = props.showCriticalSignals
+    ? [
+        {
+          high_critical:
+            props.progressBarData
+              .filesWithHighlyCriticalExternalAttributionsCount,
+          medium_critical:
+            props.progressBarData
+              .filesWithMediumCriticalExternalAttributionsCount,
+          rest:
+            props.progressBarData.filesWithOnlyExternalAttributionCount -
+            (props.progressBarData
+              .filesWithHighlyCriticalExternalAttributionsCount +
+              props.progressBarData
+                .filesWithMediumCriticalExternalAttributionsCount),
+        },
+      ]
+    : [
+        {
+          attribution: props.progressBarData.filesWithManualAttributionCount,
+          preselected:
+            props.progressBarData.filesWithOnlyPreSelectedAttributionCount,
+          signals: props.progressBarData.filesWithOnlyExternalAttributionCount,
+          rest:
+            props.progressBarData.fileCount -
+            (props.progressBarData.filesWithManualAttributionCount +
+              props.progressBarData.filesWithOnlyPreSelectedAttributionCount +
+              props.progressBarData.filesWithOnlyExternalAttributionCount),
+        },
+      ];
+
   return (
-    <MuiBox sx={props.sx}>
-      <MuiTooltip
-        title={
+    <MuiBox sx={{ ...props.sx, ...classes.bar }} aria-label={'ProgressBar'}>
+      <ResponsiveBar
+        data={data}
+        keys={Object.keys(data[0])}
+        margin={{ top: 5, right: 1, bottom: 5, left: 6 }}
```

Comment:
these margins are asymmetrical and deviate from the multiples-of-4 rule. i think there is no reason to break the rules and introduce asymmetry here:
```suggestion
        margin={{ top: 4, bottom: 4 }}
```